app/Http/Controllers/Teams/TeamsController.php

```diff
@@ -231,11 +211,11 @@ public function joined ()
      */
     public function members ()
     {
-        $query = $this->filterTeamMembers(request()->team_id);
+        $team = Team::query()->find(request()->team_id);
```

Comment:
We can use `Team::find($teamId)` here, no need to use query()

Reply:
True, but using `query()` allows the IDE to provide type hints and suggestions on the `$team` object, whereas using `find()` directly doesn't, and highlights the following method calls in yellow.